.talismanrc

```diff
@@ -180,6 +180,18 @@ fileignoreconfig:
 - filename: Taskfile.yaml
   checksum: 592e19dbbafe04c136aa73a4921ac04b2d0473e654416b08d09e81d7544b8b09
 
+fileignoreconfig:
```

Comment:
I didn't notice before but we've ended up with duplicate `fileignoreconfig` blocks in here. We only really need one with all ignorable files as items in the list.

Also it is often preferable to use the `allowed_patterns`, instead of a checksum, provided the file has a simple false positive pattern. Just saves updating the talismanrc every time we make a change. But whatever.